### ISSUE TITLE
Quickfix for unused parameter inspection

### ISSUE THREAD

**Issue by dlsniper** [go-lang-plugin-org/go-lang-idea-plugin#2470]
After merging: https://github.com/go-lang-plugin-org/go-lang-idea-plugin/pull/2297 the plugin should also gain the ability to remove the unused parameters from a function.
Caveat:
The inspection doesn't check if the function is used to implement a method defined on an interface / type in a package that's not defined in the current project. In this case the inspection should be updated to support this case and the quickfix should not be offered to the user.

**Comment by horkhe** [comment]
Please make sure the does not highlight a parameter as unused at all, if the function is implemented to meet an interface. Or should I create another issue for that?

**Comment by dlsniper** [comment]
At the moment we don't check against implementing an interface, it's something the plugin doesn't have yet. That's why I've mentioned that in the body of this issue. However, from the other point of view, an unused parameter is still an unused parameter so if you are the owner of the interface (the interface is defined in the current package, maybe even current project?) then it could make sense to offer to remove that as well (not just rename the parameter to `_`).
Nonetheless, thanks for pointing it out, hope it's more clear now. As for checking against implementing an interface, not sure if that needs its own ticket or not (maybe it should get one to have it as a reminder?).

**Comment by zolotov** [comment]
@horkhe indeed, it was bad idea, I'll remove checking method-parameters in the next alpha

**Comment by zolotov** [comment]
@dlsniper it's not about "checking against implementing an interface", it's about wrong inspection implementation for method parameters
